### PR TITLE
Qiime wrappers

### PR DESCRIPTION
**Pull request by bebatut**
Hello, 

I worked on several wrappers.

However, I have some issues:
- `align_seq`: issue with non standard methods
- `pick_open_reference_otus`, `core_diversity_analyses` or any tools using `biom` command: `ImportError: No module named pyqi.core.interface`  [submitted issue](https://github.com/galaxyproject/tools-iuc/issues/551)
- Test all wrappers with txt files because biom format not in default galaxy environment

Bérénice

### CODE REVIEW

**Comment by bgruening** [comment]
Wow!!!!
@bebatut biom should be in 16.01 afaik: https://github.com/galaxyproject/galaxy/blob/dev/config/datatypes_conf.xml.sample#L440

File: tools/qiime/align_seqs.xml [hunk 1]
```diff
@@ -1,89 +1,128 @@
...
+    ]]>
+    </version_command>
 
-#if str($pairwise_alignment_method) != 'None':
- -a $pairwise_alignment_method
-#end if
+    <command>
+<![CDATA[
+        align_seqs.py 
+            -i $input_fasta_fp
```
Comment (blankenberg) [inline tools/qiime/align_seqs.xml]: quotes

**Comment by blankenberg** [comment]
xref: https://github.com/galaxyproject/tools-iuc/pull/431

**Comment by bgruening** [comment]
Thanks a lot for working on this and sorry for my long delay!
I have created a conda package for it and hopefully we can now run automatic tests against it. I will rebase the original branch.

**Pull request merged**